src/components/App/App.tsx

```diff
@@ -23,30 +24,36 @@ const App = () => {
     thursday: false,
     friday: false,
     saturday: false
-  })
+  });
   const [bio, setBio] = useState('');
+  const [menuOpen, setMenuOpen] = useState(false);
 
-  const formProps = {name, setName, username, setUsername, email, 
+  const formProps = {name, setName, email, 
     setEmail, page, setPage, make, setMake, model, setModel, year, setYear, origin, 
     setOrigin, destination, setDestination, days, setDays, departTime, setDepartTime, 
     password, setPassword, bio, setBio
   };
 
+  const toggleMenu = () => {
+    console.log('es')
```

Comment:
Let's remember to take out our console logs!

Reply:
Good catch